calico-enterprise/operations/monitor/metrics/recommended-metrics.mdx

```diff
@@ -0,0 +1,574 @@
+| Threshold breach symptoms        | N/A                                                          |
+| Threshold breach recommendations | If this metrics consistently reports a zero value over an acceptable period of time, you can consider removing the policy rule. |
+| Priority Level                   | Optional.                                                    |
+
+### Connections per policy rule
+
+| Connections per policy rule      |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | <code>cnx_policy_rule_connections</code>                                |
+| Example value                    | <code>cnx_policy_rule_connections{**endpoint**="calico-metrics-port", **instance**="ip-10-0-1-20.ca-central-1.compute.internal", **job**="calico-node-metrics", **namespace**="calico-system", **pod**="calico-node-qzpkt", **policy**="es-kube-controller-access", **rule_direction**="egress", **rule_index**="0", **service**="calico-node-metrics", **tier**="allow-tigera", **traffic_direction**="outbound"}</code> |
+| Explanation                      | Number connections handled by {{prodname}} policy rules.              |
+| Threshold value recommendation   | This metric is similar to *Traffic per Rule* but this deals more with **flow** monitoring. This metric should usually be non-zero. A zero value indicates that the rule is not matching any packets and could be surplus to requirements. |
+| Threshold breach symptoms        | N/A                                                          |
+| Threshold breach recommendations | If this metrics consistently reports a zero value over an acceptable period of time, this policy rule can be considered for removal. |
+| Priority Level                   | Optional.                                                    |
+
+## Typha general metrics
+
+### Datastore cache size
+
+| Datastore cache size             |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | **Individual syncer values**:<br /><code>max by (instance) (typha_cache_size{syncer="bgp"})</code> <br /><code>max by (instance) (typha_cache_size{syncer="dpi"})</code><br /><code>max by (instance) (typha_cache_size{syncer="felix"})</code><br /><code>max by (instance) (typha_cache_size{syncer="node-status"})</code><br /><code>max by (instance) (typha_cache_size{syncer="tunnel-ip-allocation"})</code><br /><br />**Sum of all syncers**:<br /><code>sum by (instance)</code> <code>(typha_cache_size)</code><br /><br />**Largest syncer**:<br /><code>max by (instance)</code> <code>(typha_cache_size)</code> |
```

Comment:
> Sum of all syncers
> Largest syncer:

It's really the sum of all cache sizes.  Each syncer type has a cache and it's the cache that we're measuring, not the syncer.

Reply:
Added.